<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.0" language="fr_FR">
<context>
    <name>AllowProcess</name>
    <message>
        <location filename="logic/processes/allowprocess.cpp" line="112"/>
        <source>Network error </source>
        <translation>Erreur réseau </translation>
    </message>
    <message>
        <location filename="logic/processes/allowprocess.cpp" line="134"/>
        <source>Parsing error :</source>
        <oldsource>Parsing error :
</oldsource>
        <translation>Erreur de parsage :</translation>
    </message>
    <message>
        <location filename="logic/processes/allowprocess.cpp" line="141"/>
        <source>Unexpected problem :</source>
        <oldsource>Unexpected problem :
</oldsource>
        <translation>Problème inattendu :</translation>
    </message>
    <message>
        <location filename="logic/processes/allowprocess.cpp" line="167"/>
        <source>Configuration file does not exist.</source>
        <translation>Le fichier de configuration n&apos;existe pas.</translation>
    </message>
    <message>
        <location filename="logic/processes/allowprocess.cpp" line="171"/>
        <source>Configuration file cannot be opened.</source>
        <translation>Le fichier de configuration ne peut pas s&apos;ouvrir.</translation>
    </message>
    <message>
        <location filename="logic/processes/allowprocess.cpp" line="175"/>
        <source>Unknown problem</source>
        <translation>Problème inconnu</translation>
    </message>
</context>
<context>
    <name>LaunchingControl</name>
    <message>
        <location filename="logic/controls/launchingcontrol.cpp" line="160"/>
        <source>Reyn Tweets was denied.</source>
        <translation>eyn Tweets n&apos;a pas été autorisé.</translation>
    </message>
</context>
<context>
    <name>LaunchingProcess</name>
    <message>
        <location filename="logic/processes/launchingprocess.cpp" line="57"/>
        <location filename="logic/processes/launchingprocess.cpp" line="221"/>
        <source>Configuration file does not exist.</source>
        <translation>Le fichier de configuration n&apos;existe pas.</translation>
    </message>
    <message>
        <location filename="logic/processes/launchingprocess.cpp" line="61"/>
        <location filename="logic/processes/launchingprocess.cpp" line="225"/>
        <source>Configuration file cannot be opened.</source>
        <translation>Le fichier de configuration ne peut pas s&apos;ouvrir.</translation>
    </message>
    <message>
        <location filename="logic/processes/launchingprocess.cpp" line="65"/>
        <source>Configuration cannot be loaded.</source>
        <translation>Le fichier de configuration ne se charge pas.</translation>
    </message>
    <message>
        <location filename="logic/processes/launchingprocess.cpp" line="70"/>
        <location filename="logic/processes/launchingprocess.cpp" line="230"/>
        <source>Unknown problem</source>
        <translation>Problème inconnu</translation>
    </message>
    <message>
        <location filename="logic/processes/launchingprocess.cpp" line="151"/>
        <source>The user was not the right one.</source>
        <translation>L&apos;utilisateur n&apos;est pas le bon.</translation>
    </message>
    <message>
        <location filename="logic/processes/launchingprocess.cpp" line="161"/>
        <source>Tokens for authentication to Twitter were wrong.</source>
        <translation>Vos identifiants à Twitter sont faux.</translation>
    </message>
    <message>
        <location filename="logic/processes/launchingprocess.cpp" line="169"/>
        <source>You reach the authentication rate.</source>
        <translatorcomment>Traduction vague.</translatorcomment>
        <translation>Vous avez trop appelé Twitter.</translation>
    </message>
    <message>
        <location filename="logic/processes/launchingprocess.cpp" line="176"/>
        <source>Twitter is down.</source>
        <translation>Twitter est KO.</translation>
    </message>
    <message>
        <location filename="logic/processes/launchingprocess.cpp" line="183"/>
        <source>Unknown prolem.</source>
        <translation>Problème inconnu.</translation>
    </message>
    <message>
        <location filename="logic/processes/launchingprocess.cpp" line="190"/>
        <source>Unexpected result.</source>
        <translation>Résultat inattendu.</translation>
    </message>
</context>
<context>
    <name>OAuthProcess</name>
    <message>
        <location filename="logic/processes/oauthprocess.cpp" line="105"/>
        <source>Callback URL not confirmed.</source>
        <translatorcomment>Trouver mieux que &quot;callback&quot;</translatorcomment>
        <translation>URL callback non confirmée.</translation>
    </message>
    <message>
        <location filename="logic/processes/oauthprocess.cpp" line="117"/>
        <location filename="logic/processes/oauthprocess.cpp" line="188"/>
        <location filename="logic/processes/oauthprocess.cpp" line="290"/>
        <location filename="logic/processes/oauthprocess.cpp" line="378"/>
        <source>Network error </source>
        <translation>Erreur réseau </translation>
    </message>
    <message>
        <location filename="logic/processes/oauthprocess.cpp" line="137"/>
        <location filename="logic/processes/oauthprocess.cpp" line="210"/>
        <location filename="logic/processes/oauthprocess.cpp" line="313"/>
        <location filename="logic/processes/oauthprocess.cpp" line="400"/>
        <source>Parsing error:</source>
        <oldsource>Parsing error:
</oldsource>
        <translation>Erreur de parsage :</translation>
    </message>
    <message>
        <location filename="logic/processes/oauthprocess.cpp" line="144"/>
        <location filename="logic/processes/oauthprocess.cpp" line="217"/>
        <location filename="logic/processes/oauthprocess.cpp" line="320"/>
        <location filename="logic/processes/oauthprocess.cpp" line="406"/>
        <source>Unexpected problem:</source>
        <oldsource>Unexpected problem:
</oldsource>
        <translation>Problème inattendu :</translation>
    </message>
    <message>
        <location filename="logic/processes/oauthprocess.cpp" line="278"/>
        <source>Unexpected redirection. Process aborted.</source>
        <oldsource>Unexpected redirection. Process aborted.
</oldsource>
        <translation>Redirection innatendue. Fin du processus.</translation>
    </message>
</context>
</TS>
